addons/web/static/src/views/form/form_controller.scss

```diff
@@ -1129,24 +1096,14 @@
     flex-flow: row nowrap !important;
 
     .o_form_view_container {
-        width: $o-form-sheet-min-width + 2 * $o-horizontal-padding;
+        width: 1px; // List view needs a width value to recompute the size correctly
```

Comment:
This changes seems to break the form view of Project Sharing. (I added a point about that in your pad). 🤔